graphql/student/fields.ts

```diff
@@ -73,6 +77,23 @@ export class ExtendFieldsStudentResolver {
         });
     }
 
+    @FieldResolver((type) => String)
+    @Authorized(Role.ADMIN, Role.OWNER)
+    async instantCertificate(@Root() student: Required<Student>, @Arg('lang') lang: string) {
```

Comment:
Maybe rather as a mutation - its quite heavy to render a PDF in a field resolver ...